### ISSUE TITLE
What about "testdata"

### ISSUE THREAD

**Issue by Niondir** [golang-standards/project-layout#3]
It seems common to have a "testdata" directory for additional test data. Any specific reason why it's not included in this repo?

**Comment by q-cloudlock** [comment]
The /test directory is the place to store test data in addition to any extra test tools you need, so you might have /test/data and /test/tools or some other subdirectory structure.

**Comment by kcq** [comment]
The /test directory is the place to store test data in addition to any extra test tools you need, so you might have /test/data and /test/tools or some other subdirectory structure.

**Comment by kcq** [comment]
@coreydaley nice example! The same OpenShift repo includes a number of other common project layout patterns: /cmd (https://github.com/openshift/origin/tree/master/cmd), /pkg (https://github.com/openshift/origin/tree/master/pkg), /tools (https://github.com/openshift/origin/tree/master/tools), /api (https://github.com/openshift/origin/tree/master/api) and /docs (https://github.com/openshift/origin/tree/master/docs).

**Comment by kcq** [comment]
True, `testdata` is special, so you can make it `test/testdata` if you want to take advantage of it :-)

**Comment by adamdecaf** [comment]
Would y'all accept a doc PR mentioning how `testdata` is special?

**Comment by scirlig-ellation** [comment]
> fwiw, a `testdata` directory is special, because it will be skipped by go commands like `go install ./...`. `test/data` does not have this effect.

> fwiw, a `testdata` directory is special, because it will be skipped by go commands like `go install ./...`. `test/data` does not have this effect.

I don't think you're right. Take a look inside `$GOPATH/bin` after running `go install ./...` and testdata contains some files with `package main` declaration. I'm still trying to understand what's so special about this dir
